mmv1/products/netapp/Volume.yaml

```diff
@@ -618,3 +618,64 @@ properties:
     description: |
       Total hot tier data rounded down to the nearest GiB used by the volume. This field is only used for flex Service Level
     output: true
+  - name: 'cacheParameters'
+    type: NestedObject
+    description: |-
+      Cache parameters for the volume.
+    properties:
+      - name: 'peerVolumeName'
+        type: String
+        description: |
+          Required. Name of the origin volume for the cache volume.
+      - name: 'peerClusterName'
+        type: String
+        description: |
+          Required. Name of the origin volume's ONTAP cluster.
+      - name: 'peerSvmName'
+        type: String
+        description: |
+          Required. Name of the origin volume's SVM.
+      - name: 'peerIpAddresses'
+        type: Array
+        description: |
+          Required. List of IC LIF addresses of the origin volume's ONTAP cluster.
+        item_type:
+          type: String
+      - name: 'enableGlobalFileLock'
+        type: Boolean
+        description: |
+          Optional. Field indicating whether cache volume as global file lock enabled.
+      - name: 'peeringCommandExpiryTime'
+        type: String
+        description: |
+          Optional. Expiration time for the peering command to be executed on user's ONTAP. A timestamp in RFC3339 UTC "Zulu" format. Examples: "2023-06-22T09:13:01.617Z".
+        default_from_api: true
```

Comment:
I am curious the reason to use `default_from_api: true` here.
Will API return the default value in API response if this field is not provided in API CREATE request?